### ISSUE TITLE
Verify deployment not working for Kubernetes 1.18

### ISSUE THREAD

**Issue by MKDonnelly** [FairwindsOps/rok8s-scripts#378]
I encountered an issue when trying to create a new server with Kubernetes 1.18 and rok8s-scripts. I copied a set of configurations from an existing deployment that is working, however, when I go to deploy it, I received this message:
```
observedGeneration ({}
1) should be >= generation (1).
/usr/local/bin/verify-deployment: line 11: [: too many arguments
```

I noticed that this was caused by [line 14 in your rok8s files](https://github.com/FairwindsOps/rok8s-scripts/blob/master/bin/verify-deployment) as shown here
`OBS_GEN=$(kubectl get deployment "${1}" --namespace="${2:=default}" -o yaml | grep observedGeneration: | awk '{print $2}')`

On our current deployments (non-Kubernetes 1.18), the output of the first two parts minus the "| awk '{print $2}'") is this

`observedGeneration: 1`

However, in Kubernetes 1.18, a managedField entry is also displayed which sort of looks like a default config (example of what this looks like is [here](https://github.com/kubernetes/kubernetes/issues/89080)) with many of the values set to {}. As a result, that line ends up with this
```
      f:observedGeneration: {}
  observedGeneration: 1
```

Which means the awk statement is parsing out

```
{}
1
```

instead of

`1`

as it is supposed to. This is exactly what is happening in the error I see. I found two issues on github about the extra managedField part ([here](https://github.com/kubernetes/kubernetes/issues/89080) and [here](https://github.com/kubernetes/kubernetes/issues/90066)) and there is no good way for us to turn it off. Our existing server deployments work fine, it's just that when creating a new server that I hit this issue.

**Comment by sudermanjr** [comment]
Nice catch, and thanks for the PR! I didn't actually think anyone used the raw manifest deploys anymore, so it's good to see some people using the tool.

**Comment by MKDonnelly** [comment]
Sure thing. I'm still working on the PR to verify that it works on our new deployment. I'm also going to test it on our other deployments to verify there are no regressions.

**Comment by sudermanjr** [comment]
Fixed by #379
